### ISSUE TITLE
about smooth label

### ISSUE THREAD

**Issue by mk123qwe** [CoinCheung/pytorch-loss#4]
if i choose smooth label ,what loss function do i choose?
in ptorch, celoss only support traget.long()

**Comment by CoinCheung** [comment]
Sorry for replying so late. Maybe you have figured it out yourself, you can use label smooth just like you use cross-entropy-loss of pytorch, only to be sure to add the args of `smooth` which controls the smooth label.
